pmm/pmm2-ovf.groovy

```diff
@@ -31,11 +35,38 @@ pipeline {
             }
         }
 
-        stage('Build Image') {
+        stage('Build Image Release Candidate') {
+            when {
+                expression { env.RELEASE_CANDIDATE == "yes" }
+            }
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                    sh """
+                        packer build \
+                        -var 'pmm_client_repos=original testing' \
+                        -var 'pmm_client_repo_name=percona-testing-x86_64' \
+                        -var 'pmm2_server_repo=testing' \
+                        -only virtualbox-ovf -color=false packer/pmm2.json \
+                            | tee build.log
+                    """
+                }
+                sh 'ls */*.ova | cut -d "/" -f 2 > IMAGE'
```

Comment:
Better to replace by `find` instead of `ls + cut` construction

Reply:
I haven't really changed anything on this, just using the same way as we do for current builds.